incubator/aws-autoscaling-exporter/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: aws-autoscaling-exporter
+home: https://banzaicloud.com
+sources:
+  - https://github.com/banzaicloud/charts/
```

Comment:
This one is invalid: the right one is https://github.com/banzaicloud/banzai-charts